### ISSUE TITLE
Add Windows Support

### ISSUE THREAD

**Issue by robmorgan** [cakephp/phinx#16]
Phinx should work on Windows and be installable via Composer.

**Comment by gayancng** [comment]
I am a Windows user.I installed  Phinx libs to my application.
App root : C:\xampp\htdocs\verdict\mobile-booking
plugin root :  C:\xampp\htdocs\verdict\mobile-booking\plugin

When i run this command it's getting a warning like this.

C:\xampp\htdocs\verdict\mobile-booking\plugin>php vendor/bin/phinx

SRC_DIR="`pwd`"
cd "`dirname "$0"`"
cd "../robmorgan/phinx/bin"
BIN_TARGET="`pwd`/phinx"
cd "$SRC_DIR"
"$BIN_TARGET" "$@"

Kindly please help me to fix this issue.

**Comment by rquadling** [comment]
TL;DR
```vendor\bin\phinx``` is NOT a PHP file. So don't use PHP against it. Just use ```vendor\bin\phinx```.


```vendor\bin\phinx``` is a shell script file for unix.

But, if you look, you'll also see ```vendor\bin\phinx.bat``` as Composer on Windows automatically provides the helper script.

Now. Assuming you've not been buggering around with your environment variables, you can simply run ```vendor\bin\phinx```.

Now you may be thinking "Hold on, that'll try and run the unix shell script and I'm on Windows and that's not going to work and this guy doesn't know what he is talking about!".

And in response, I'll gently snigger with a slightly smug sense of superiority.

What actually happens is that when you type a command at the command line, CMD has to work out what to execute. There are several steps it goes through.

One of them is to try and find a file that is "executable". Hmm. Or rather has an extension that Windows has a registered executor for.

The extensions are set up in an environment variable called PATHEXT (you can see the value by typing ```SET PATHEXT```). Mine looks like this ```PATHEXT=.COM;.EXE;.BAT;.CMD;.PHP;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.RB;.RBW;.PY```

So, when you typed ```vendor\bin\phinx```, internally, it went looking for the following files ...

```
vendor\bin\phinx.exe
vendor\bin\phinx.com
vendor\bin\phinx.bat
```

When it found ```vendor\bin\phinx.bat``` Windows then went and found the appropriate executor - which in this case is the shell itself (sort of a clever autodetection by the shell I think).

So. If you are interested, you can make PHP scripts work the same way. AND you can make then run without having to type PHP to load them AND you can make PHP scripts act as filters.

As an example, I have a script that strips out ANSI codes from output (normally used for scripts that don't have a --no-ansi option).

I can use it by ```Command --that --makes --ansi --codes | noansi```.

Rather than ```Command --that --makes --ansi --codes | C:\php5\php.exe -f C:\Utils\noansi.php```.

For all the juicy help read all about it in http://php.net/manual/en/install.windows.commandline.php
